gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/storage/LocalStorageService.java

```diff
@@ -43,6 +43,10 @@ public void deleteFile(String filename) {
         }
     }
 
+    public String getFileLocation(String filename) {
+        return filename;
```

Comment:
yeah - "downloadFile" and "deleteFile" bot use filename as the file location for local storage so I figured this suffices when using local storage and needed an implementation of getFileLocation to fully implement StorageService.

The only thing I was thinking it could be is that it needs to remain `file://localhost/{filename}`, so depends what we need the result to be.

Hopefully that makes sense? Obviously open to better suggestions :)